docs/integrate_ultra.md

```diff
@@ -0,0 +1,40 @@
+# Integration with Ultra
```

Comment:
This file has to be added to mkdocs.yml file so that it appears in the navigation
See : https://www.mkdocs.org/user-guide/configuration/#documentation-layout

Reply:
Done.